src/pages/profiles/ProfileUsedByDefaultProject.tsx

```diff
@@ -32,17 +35,7 @@ const ProfileUsedByDefaultProject: FC<Props> = ({
               <i className="u-text--muted no-instances">No instances</i>
             )}
             {project.instances.length > 0 && (
-              <ExpandableList
-                items={project.instances.map((instance) => (
-                  <div
-                    key={instance.name}
-                    className="u-truncate list-item"
-                    title={instance.name}
-                  >
-                    <InstanceLink instance={instance} />
-                  </div>
-                ))}
-              />
+              <ViewAllInstancesBtn profile={profile} project={projectName} />
```

Comment:
This is the wrong project when referencing another one. I think it should be `project.name`, and we don't need to pass in the `projectName` here.